core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -208,12 +210,20 @@ private String getDetectedKeywords(Collection<SearchResult> searchResults, Strin
                  .forEach(s -> {
                    Matcher matcher = PATTERN.matcher(s);
                    while (matcher.find()) {
-                     String keyword = dlpKeywordsList.stream().filter(key -> matcher.group(1).contains(key)).findFirst().orElse(null);
+                     String keyword = dlpKeywordsList.stream().filter(key -> removeAccents(matcher.group(1)).contains(removeAccents(key))).findFirst().orElse(null);
                      if (keyword != null && !keyword.isEmpty() && !detectedKeywords.contains(keyword)) {
                        detectedKeywords.add(keyword);
                      }
                    }
                  });
     return detectedKeywords.stream().collect(Collectors.joining(", "));
   }
+
+  private String removeAccents(String string) {
+    if (StringUtils.isNotBlank(string)) {
+      string = Normalizer.normalize(string, Normalizer.Form.NFD);
+      string = string.replaceAll("[\\p{InCombiningDiacriticalMarks}]", "");
```

Comment:
are you sure of this line ?

Reply:
Yes

Reply:
Can you explain where it comes from ?

Reply:
The \p{InCombiningDiacriticalMarks} will match all such diacritic codes and we will replace them with their deAccented counterparts followed by their combining diacritics.
we have already used it in other places https://github.com/Meeds-io/social/blob/d72cad054a333238e99f0e638f271fd9cc94555c/component/core/src/main/java/org/exoplatform/social/core/jpa/search/ProfileSearchConnector.java#L443